lib/rules/prefer-user-event.ts

```diff
@@ -49,17 +49,14 @@ export const MappingToUserEvent: Record<string, UserEventMethodsType[]> = {
 };
 
 function buildErrorMessage(fireEventMethod: string) {
-  const allMethods = MappingToUserEvent[fireEventMethod].map(
-    (method: string) => `userEvent.${method}()`
+  const userEventMethods = MAPPING_TO_USER_EVENT[fireEventMethod].map(
+    (methodName) => `userEvent.${methodName}`
   );
-  const { length } = allMethods;
 
-  const init = length > 2 ? allMethods.slice(0, length - 2).join(', ') : '';
-  const last = `${length > 1 ? ' or ' : ''}${allMethods[length - 1]}`;
-  return `${init}${last}`;
+  return userEventMethods.join(', ').replace(/, ([a-zA-Z.]+)$/, ', or $1');
```

Comment:
I'd like to just use `Intl.ListFormat` here, but it's available from node v13 so we can't stick to it until we drop node v10 and node v12 compatibility 😅